### ISSUE TITLE
I'm trying to train with KSS Model

### ISSUE THREAD

**Issue by Zegalryang** [TensorSpeech/TensorFlowTTS#415]
Hello everyone!

I'm trying to train KSS model following instruction.
after train, it can not synthesis. (but your pretrained model is working)

1. tensorflow-tts-preprocess --rootdir ./kss --outdir ./dump_kss --config preprocess/kss_preprocess.yaml --dataset kss

2. tensorflow-tts-normalize --rootdir ./dump_kss --outdir ./dump_kss --config preprocess/kss_preprocess.yaml --dataset kss

3. python examples/tacotron2/extract_duration.py   --rootdir ./dump_kss/train/   --outdir ./dump_kss/train/durations/   --checkpoint tacotron2-100k.h5  --use-norm 1   --config ./examples/tacotron2/conf/tacotron2.kss.v1.yaml   --batch-size 32

4. python examples/tacotron2/extract_duration.py   --rootdir ./dump_kss/valid/   --outdir ./dump_kss/valid/durations/   --checkpoint tacotron2-100k.h5  --use-norm 1   --config ./examples/tacotron2/conf/tacotron2.kss.v1.yaml   --batch-size 32
* tracotron2-100k.h5 is pretrained KSS model

5. python examples/fastspeech2/train_fastspeech2.py   --train-dir ./dump_kss/train/   --dev-dir ./dump_kss/valid/   --outdir ./examples/fastspeech2/exp/train.kss.fastspeech2.v1/   --config ./examples/fastspeech2/conf/fastspeech2.kss.v1.yaml   --use-norm 1   --f0-stat ./dump_kss/stats_f0.npy   --energy-stat ./dump_kss/stats_energy.npy   --mixed_precision 1   --resume ""

did I do something wrong?

**Comment by dathudeptrai** [comment]
> 3\. python examples/tacotron2/extract_duration.py   --rootdir ./dump_kss/train/   --outdir ./dump_kss/train/durations/   --checkpoint tacotron2-100k.h5  --use-norm 1   --config ./examples/tacotron2/conf/tacotron2.kss.v1.yaml   --batch-size 32

can you provide tensorboard ?

**Comment by Zegalryang** [comment]
is this right?

![Untitled](https://user-images.githubusercontent.com/2080839/101312823-11380300-3898-11eb-8e28-1fee93c7ea64.png)

**Comment by dathudeptrai** [comment]
> is this right?
> 
> ![Untitled](https://user-images.githubusercontent.com/2080839/101312823-11380300-3898-11eb-8e28-1fee93c7ea64.png)

ur durations loss is very high, it should be < 0.1. BTW, can you give me a full tensorboard figure ?

cc : @crux153

**Comment by Zegalryang** [comment]
of course.

would i send you this file 'events.out.tfevents.xxxx.v2' ?

**Comment by dathudeptrai** [comment]
> of course.
> 
> would i send you this file 'events.out.tfevents.xxxx.v2' ?

just need to capture full screen :D

**Comment by Zegalryang** [comment]
thanks!!

![fullscreen](https://user-images.githubusercontent.com/2080839/101313494-9243ca00-3899-11eb-9238-64816664fad8.png)

**Comment by crux153** [comment]
I don't think that tensorboard is normal. It should look like this as in #130.

![](https://user-images.githubusercontent.com/10908841/88132963-d51f4100-cc1b-11ea-8efc-9e24cfc73c93.png)

It's strange since I don't see any problem with your commands, except that I **didn't** used mixed precision. Maybe try again without mixed precision?

I'm now training tacotron2, and will try to train fastspeech2 again with latest code. Let's see if there's any problem.
